js/default-table-data/layouts-table-config.js

```diff
@@ -53,7 +53,7 @@ window.flListLayoutTableConfig = {
       'Image': 'https://cdn.fliplet.com/apps/2659/de1ac7174a201cff518f13b38a293d30363-001-3021.jpg',
       'Categories': 'Enterprise software',
       'Date': moment().locale('en').subtract(3, 'months').format('YYYY-MM-DD'),
-      'Content': '<p>Most enterprises now agree that their bottom line could benefit from bespoke apps. But which ones are they building and where do they think the real value lies?</p><p>Fliplet works with many of the world’s industry leading businesses. We’ve crunched the data to identify the trends in the world of enterprise app building. In this report we’ll share some of the findings and talk through a few case studies that are at the cutting edge</p><h3>Revenue generation or cost savings</h3><p>Technology in the workplace is often thought of as a way to streamline process, reducing friction and ultimately creating cost savings. However it is actually the revenue generating functions that are making the widest use of apps. Sales and marketing teams are using apps to reach out to new clients and engage more with existing ones. They tend to be amongst the most prolific app builders - producing an a new app every couple of months</p><p>To learn more about enterprise app trends - visit <a href=“https://fliplet.com”>Fliplet.com</a></p>'
```

Comment:
The old version had `“` and `”` in the `<a href>` at the end of the line, affecting how the links are working.